### ISSUE TITLE
Abyssus Essential

### ISSUE THREAD

**Issue by SideMatter** [1kc/razer-macos#241]
Hello, I have the Abyssus Essential and it works sometimes. let me explain. If I change the setting in the app then un plug and replug it changes the light 25% of the time. Not sure if that's helpful, but if I can help debug in any way id be more then happy to. I have Abyssus Essential, Golithus, and Cynosa chroma

**Comment by WolfBoy11YT** [comment]
yes please fix this i kinda have the same problem but the problem is the wave doesn't work
